### DISCUSSION TITLE
Avoid confirmation window on `clone-in-kitty`

### DISCUSSION THREAD

**Discussion by anjos** [kovidgoyal/kitty#6036] [Q&A] [answered]
I wonder if it is possible to either pass a flag, or set a configuration variable to avoid the confirmation windows that is popped-up every time the command `clone-in-kitty` is used.  I searched around, but did not find anything useful.  Thanks!

**Comment by page-down** [comment, marked as answer]
> ... I searched around, ...

I had no problem finding the relevant options by searching for `clone-in-kitty`, just above `clone_source_strategies`.

```conf
allow_cloning yes
```

https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.allow_cloning

You can also generate a commented default configuration file for the version of kitty you are running and there is no need to search through the website.

**Reply by anjos** [reply]
Thanks, @page-down.  I apologise for the triviality.
